tasks/serializers.py

```diff
@@ -0,0 +1,56 @@
+from rest_framework import serializers
+from .models import Task
+
+
+class CreateTaskSerializer(serializers.ModelSerializer):
+
+    share_link = serializers.CharField(default=None)
+    share_text = serializers.CharField(default=None)
+    share_img = serializers.ImageField(default=None)
+
+    class Meta:
+        model = Task
+        fields = [
+            'social_media_platform', 'description', 'share_type', 'share_link', 'share_text', 'share_img']
+
+    def save(self, user, org):
+
+        valid_data = self.validated_data
+
+        #Check for presence of Link if share type is link and then create respective task.
+        if valid_data['share_type'] == 'LINK':
+            if valid_data['share_link'] is None:
+                raise serializers.ValidationError({'Share_Link':'Share link is required'})
```

Comment:
```suggestion
                raise serializers.ValidationError({'share_Link':'Share link is required'})
```
this is because having different cases for input and response might confuse frontend developers

Reply:
Got it 🙌